kiwi/defaults.py

```diff
@@ -70,12 +70,20 @@ def get_xz_compression_options(self):
         ]
 
     @classmethod
-    def is_obs_worker(self):
+    def is_buildservice_worker(self):
         # the presence of /.buildenv on the build host indicates
         # we are building inside of the open buildservice
         return os.path.exists('/.buildenv')
 
     @classmethod
+    def get_obs_download_server_url(self):
+        """
+        The default download server url hosting the public open
+        buildservice repositories
+        """
+        return 'http://download.opensuse.org'
+
```

Comment:
I believe we could include the `repositories` part in the URL. Well, this is a matter of coordinating with he buildservice guys, but form Adrian example in email thread, I would assume they give the base URL including `/repositories`. I would like to stick as much as possible to the data provided by the buildservice, without having to parse it and just use it as is.

```
# osc api /configuration  | grep download_url
<download_url>http://download.opensuse.org/repositories</download_url>
```